cmd/gitops-server/cmd/cmd.go

```diff
@@ -113,6 +113,7 @@ func NewCommand() *cobra.Command {
 	cmd.Flags().DurationVar(&options.OIDC.TokenDuration, "oidc-token-duration", time.Hour, "The duration of the ID token. It should be set in the format: number + time unit (s,m,h) e.g., 20m")
 	cmd.Flags().StringVar(&options.OIDC.ClaimsConfig.Username, "oidc-username-claim", auth.ClaimUsername, "JWT claim to use as the user name. By default email, which is expected to be a unique identifier of the end user. Admins can choose other claims, such as sub or name, depending on their provider")
 	cmd.Flags().StringVar(&options.OIDC.ClaimsConfig.Groups, "oidc-groups-claim", auth.ClaimGroups, "JWT claim to use as the user's group. If the claim is present it must be an array of strings")
+	cmd.Flags().StringSliceVar(&options.OIDC.Scopes, "custom-oidc-scopes", auth.DefaultScopes, "Customise the requested scopes for then OIDC authentication flow - openid will always be requested")
```

Comment:
This flag doesn't seem to be connected right now, maybe we remove it for now or?

Reply:
I think it is connected, through a long line of bits.

It is passed here https://github.com/weaveworks/weave-gitops/blob/2deffd819e9d9482e033bc3cb002b62becb21aaa/cmd/gitops-server/cmd/cmd.go#L175

Reply:
Oh you're right. I was testing with mixed secret/cli config and then missed this

https://github.com/weaveworks/weave-gitops/blob/2deffd819e9d9482e033bc3cb002b62becb21aaa/pkg/server/auth/init.go#L40-L41

LGTM

Reply:
I find it a bit confusing that the secret overrides the cmd-line, but that's the existing behaviour for other options, consistency wins out :-)